src/Components/CardboardList/CardboardListItem.tsx

```diff
@@ -125,6 +125,9 @@ export const CardboardListItem = <T extends unknown>(
                         (typeof iconStart.name === 'string' ? (
                             <FontIcon
                                 iconName={iconStart.name}
+                                style={{
+                                    color: (iconStart as any).color
```

Comment:
Any chance that casting iconStart to any wouldn't have a `.color`? Should we provide a default here?

Reply:
ahh okay it was due to build error since in the type definition iconStart can either be a object (even if I added color?: string there) or react element. So my fix is not working still, I will fix it with type checking.

![image](https://user-images.githubusercontent.com/45217314/208786383-0bfe5275-e371-4957-bd3f-39622f9c36cf.png)
![image](https://user-images.githubusercontent.com/45217314/208786441-2efca76d-a09d-44e2-aa99-7a3bd3c6e4e0.png)